portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -10,6 +10,12 @@ definition {
 		Click(locator1 = "ACSettings#KEYWORD_SEND_BUTTON");
 	}
 
+	macro assertDataSourceList {
```

Comment:
Hi @calvinctvo , I would suggest you rename it for `assertDataSourceName` or `viewDataSourceName` since you are passing a single item and not a list.

Reply:
I know you are validating the data source is on the list... but renaming I think it will be clearer. What do you think?

Thanks!